Documents/customAnimal.md

```diff
@@ -0,0 +1,55 @@
+# How To Make A Custom Animal
+
+  
+
+## In this example we will be configuring a dragon animal.
+
+  
+
+### Where To Configure:
+
+The first thing you need to know is all configurations of the animal are done in console.
+
+  
+
+### Naming The Animal:
+
+The second thing you have to do is give the animal a name. The name of the animal does not effect any part of the program, so feel free to name it whatever you like!
+
+For Example,
+```console
+Animal: dragon
+```
+
+  
+
+### Setting Up Speed:
+
+Thirdly, what you need to do is input the speed of the animal, the strength of the animal, and the weight of the animal all on one line separated by a space. For example,
+
+```console
+Input speed, strength, weight: 4 17 160
+```
+
+if you want to find out how to increase or decrease speed, please refer to speedAlgorithm.md
+
+### Setting Up Defense:
+
+Once you have the speed finished you now have to configure the defensive stats of the custom animal. This part is pretty simple. If you want your animal to take no damage, you set the defense to 0. If you want your animal to take normal amount of damage, set the defense value to 1. If you want the animal to take more damage, have the defensive value be greater than 1.
+
+For example,
+
+```console
+Defense: 0.75
+```
+### Setting Up Attack:
+Setting up the attack value is very simple. The user can input any number they wish, as attack is the real damage that you do to other animals.
```

Comment:
... so higher acttack values means more damage inflicted?

Reply:
Yes

Reply:
When I ask questions like this on documentation, I'm usually alluding to "please make your documentation clearer"... so rather than answering my question in the comment, just update the documenation.